src/components/common/AgreementType.vue

```diff
@@ -22,7 +22,7 @@
     </template>
 
     <!-- EDIT SECTION -->
-    <div v-if="isTypeBcUlcCompany" :class="{ 'invalid-section': showErrorSummary && getShowErrors }">
+    <div v-if="isTypeBcUlcCompany || isTypeBcCcc" :class="{ 'invalid-section': showErrorSummary && getShowErrors }">
```

Comment:
To reflect that CCC's do not have a sample agreement the they now use the checkbox style agreement confirmation.

![Screenshot 2022-11-25 114830](https://user-images.githubusercontent.com/60455392/204050613-982f416b-ed65-451a-a09b-f2d3266a9e0e.png)

Reply:
Sounds good. Thanks for the screenshot 👍